packages/core/core/src/Transformation.js

```diff
@@ -158,6 +163,17 @@ export default class Transformation {
           },
         ]);
       }
+
+      if (existing == null) {
+        // If no existing sourcemap was found, initialize sourcesContent with
+        // the asset's filepath and its original contents.
+        asset.sourcesContent = {
```

Comment:
Why is this stored separately on the asset rather than in the source map? If there's no existing source map, should we just initialize an empty one containing only the original code?

Reply:
Because of this line: https://github.com/parcel-bundler/parcel/blob/v2/packages/core/core/src/UncommittedAsset.js#L102